SpeechResponder/ShowDiffWindow.xaml

```diff
@@ -3,17 +3,21 @@
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:avalonEdit="http://icsharpcode.net/sharpdevelop/avalonedit"
              xmlns:resx="clr-namespace:EddiSpeechResponder.Properties"
              mc:Ignorable="d"
              Title="{x:Static resx:SpeechResponder.compare_script_title}" 
              ResizeMode="CanResizeWithGrip"
              Height="600" Width="800"
              Background="#FFE5E5E5" >
-    <RichTextBox x:Name="scriptText" Background="Transparent" BorderThickness="0" Margin="5" IsReadOnly="True" FontFamily="Consolas">
-        <RichTextBox.Resources>
-            <Style TargetType="{x:Type Paragraph}">
-                <Setter Property="Margin" Value="0"/>
-            </Style>
-        </RichTextBox.Resources>
-    </RichTextBox>
+    <avalonEdit:TextEditor Margin="10" 
+                           Name="scriptView"
+                           Foreground="DarkSlateGray"
+                           Background="WhiteSmoke"
+                           FontFamily="Consolas"
+                           IsReadOnly="True"
+                           WordWrap="True"
+                           ShowLineNumbers="True"
```

Comment:
At present, the line numbers in the diff window do not match the line numbers in the edit window.
Would it be possible to adjust line numbers so that the diff window preserves the same line numbers as the edit window (perhaps by repeating the line number or omitting a repeated line number while displaying a diff)?

The diff window also adds a trailing line that isn't present in the original text.

![Untitled](https://user-images.githubusercontent.com/4382539/70199976-fdc3ec00-16c7-11ea-8e04-be4f7d611933.png)
Note: This screenshot incorporates some recommendations on margins and the toolbar grip discussed elsewhere in my comments.

Reply:
Unfortunately line numbers in AvalonEdit are an integral part of the text engine and can't be set arbitrarily. I think it's simplest to just turn off line number display in the diff window. I was adding an extra newline after the last line segment: fixed.